x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Popover/ServiceMetricList.tsx

```diff
@@ -15,7 +15,8 @@ import { i18n } from '@kbn/i18n';
 import { isNumber } from 'lodash';
 import React from 'react';
 import styled from 'styled-components';
-import { ServiceNodeMetrics } from '../../../../../server/lib/service_map/get_service_map_service_node_info';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { ServiceNodeMetrics } from '../../../../../../../../plugins/apm/server/lib/service_map/get_service_map_service_node_info';
```

Comment:
Can we avoid supressing the eslint warning by fixing the root cause? (eg. it should not be forbidden to import for that path)

Reply:
Looked into how this is used:
```ts
  const { data = {} as ServiceNodeMetrics, status } = useFetcher(
```

Is there any reason for explicitly typing  it? It should be inferred by `useFetcher`:
```ts
  const { data = {}, status } = useFetcher(
```

Reply:
Oh, nvm. I see: it's for the default value 👍